doc/docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+**Welcome to TDM!** This documentation should help you get started being productive with TDM.
+
+[[toc]]
+
+## [Guides](/guides/)
```

Comment:
The headers Guides and Developer Documentation looks odd and seems like an unfinished category not a link. I don't know if it is necessary to be here. Since it is just a pointer to a section later on.

Reply:
Moved to bullet points.